README.md

```diff
@@ -93,6 +93,74 @@ yo angular-enterprise:factory myFactory # "myFactory" is the name of factory
 yo angular-enterprise:service myService # "myService" is the name of service (singleton)
 ```
 
+## Alias
+
+Typing the command:
+
+```bash
+yo angular-enterprise
+```
+
+for each generation is cumbersome. Unfortunately, there is no way to alias the generation command through the Yeoman API. If you would like to use an alias, please follow the proceeding steps.
+**Note that you may have to do this as root (ie. sudo)**
+
+1.You can put this file anywhere, but `/usr/bin` is a good choice.Navigate to your `/usr/bin` directory and create a file titled `ae.sh`
```

Comment:
System-wide shortcuts is probably not a place to put the command. ~/.bin/ is probably better.

Also, consider placing an `ae.sh` file in the repo under `scripts/` with the code populated so that we can keep it up to date? Later, we can consider if we should simply rename the generated to "ae"